### PR TITLE
Cortx 28268 fix/improvement execution report

### PR DESCRIPTION
**Pull request by DhobaleVishal**
# Problem Statement
- Fixed corio summary reporting issues on failure/abort.
- Fixed sessions result in report.

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed
-  [x] Attach test execution logs
https://jts.seagate.com/secure/attachment/510663/reports.zip

# Review Checklist 
  Checklist for Author
-  [x] JIRA number/GitHub Issue added to PR
-  [x] PR is self reviewed
-  [x] Jira and state/status is updated and JIRA is updated with PR link
-  [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

File: corio.py [hunk 1]
```diff
@@ -237,13 +240,24 @@ def log_status(parsed_input: dict, corio_start_time: datetime.time, test_failed:
                         input_dict["RESULT_UPDATE"] = f"Passed at {pass_time}"
                     else:
                         input_dict["RESULT_UPDATE"] = "In Progress"
+                    if input_dict["RESULT_UPDATE"] == "In Progress":
+                        # Report failure and update status.
+                        if terminated_tests:
+                            if input_dict["TEST_ID"] in terminated_tests:
+                                input_dict["RESULT_UPDATE"] = "Fail"
```
Comment (priyankaborawake) [inline corio.py]: can we add the time stamp for fail/aborted

File: corio.py [hunk 2]
```diff
@@ -212,6 +214,7 @@ def log_status(parsed_input: dict, corio_start_time: datetime.time, test_failed:
         elif test_failed is None:
             status_file.write('\nTest Execution still in progress')
         else:
+            LOGGER.info(terminated_tests)
```
Comment (ketanarlulkar) [inline corio.py]: Include a statement too. That will help in searching logs.

File: corio.py (hunk 1 above)
Comment (DhobaleVishal) [reply corio.py]: Total execution time updated at line no 249.

**Review by ketanarlulkar — APPROVED** [review]

File: corio.py (hunk 2 above)
Comment (DhobaleVishal) [reply corio.py]: done

File: corio.py [hunk 3]
```diff
@@ -405,17 +417,20 @@ def main(options):
                                     process.pid, process.name)
                     terminate = True
                     terminated_tp = key
-                    test_ids = [td["TEST_ID"] for td in parsed_input[terminated_tp].values()]
+                    # Get all test id from terminated workload due to failure.
+                    for test in parsed_input[terminated_tp].values():
```
Comment (priyankaborawake) [inline corio.py:414]: what will be the test_ids in case of support bundle or heathcheck failure

Comment (DhobaleVishal) [reply corio.py:414]: Updated logic for sb and health check.

**Review by ketanarlulkar — APPROVED** [review]

**Review by priyankaborawake — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**